src/poetry/utils/env/python_manager.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import annotations
+
+import shutil
+import subprocess
+import sys
+
+from functools import cached_property
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from cleo.io.null_io import NullIO
+from cleo.io.outputs.output import Verbosity
+from poetry.core.constraints.version import Version
```

Comment:
I am not sure if we should use this as a version in the long run. We had some reports that it doesn't allow proper Python versions to be used. I am not sure if it's better to change it now or wait for the next stage of changes.